WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -123,11 +123,11 @@ struct PointOfSaleDashboardView: View {
             .frame(maxWidth: Constants.exitPOSSheetMaxWidth)
         }
         .posRootModal()
-        .sheet(isPresented: $showSupport) {
+        .posSheet(isPresented: $showSupport) {
             supportForm
                 .interactiveDismissDisabled(true)
```

Comment:
unrelated to this PR, as it happens in trunk as well: I noticed that after confirming the email alert on top of the support form, there was no obvious way to exit the form. The "Done" CTA in the navigation bar is actually the CTA to dismiss the form after checking the code, but I thought it's to submit the form as a user. Since interactive dismissal is disabled, maybe we can rename this "Done" to some dismissing action like "Cancel" and place it in the leading edge.

Reply:
Great observation. It's an easy change, I will do it and merge.